spec/models/aspace_to_folio_record_spec.rb

```diff
@@ -124,20 +130,29 @@
   end
 
   describe '#archivesspace_marc_xml_path' do
-    let(:record) { FactoryBot.create(:aspace_to_folio_record, archivesspace_instance_key: 'test_instance', repository_key: 5, resource_key: 42) }
+    let(:record) { aspace_to_folio_record }
```

Comment:
```suggestion
```
Even though it'll make the lines longer, it might be better to remove `let(:record)` and reference `aspace_to_folio_record` in this and the tests below, just to eliminate an extra layer of indirection.